pom.xml

```diff
@@ -494,6 +494,7 @@ THE SOFTWARE.
         <configuration>
           <trimStackTrace>false</trimStackTrace> <!-- SUREFIRE-1226 workaround -->
           <rerunFailingTestsCount>4</rerunFailingTestsCount>
+          <argLine>${maven-surefire-plugin.agent}</argLine>
```

Comment:
should add `${argLine}` to allow passing of arguments from the command line still (such as running with less memory than 2GB :-)